Assets/2DGamekit/Art/TileMapPalettes/Tileset/TilesetRockWaterBlockers.prefab

```diff
@@ -43,67 +43,74 @@ Tilemap:
   m_Tiles:
   - first: {x: -3, y: -1, z: 0}
     second:
+      serializedVersion: 2
       m_TileIndex: 0
       m_TileSpriteIndex: 0
       m_TileMatrixIndex: 0
       m_TileColorIndex: 0
-      m_ObjectToInstantiate: {fileID: 0}
```

Comment:
I'm planning to revert the changes to `TilesetGamekit.prefab` and `TilesetRockWaterBlockers.prefab` unless there's a reason they were changed.